### PR TITLE
483 Consolidated Case Filing Parties

### PR DESCRIPTION
**Pull request by wbyoko**
Screenshot 2019-12-12 12 38 55

### CODE REVIEW

File: shared/src/business/entities/externalDocument/ExternalDocumentInformationFactory.js [hunk 1]
```diff
@@ -262,7 +262,12 @@ ExternalDocumentInformationFactory.get = documentMetadata => {
         },
         [],
       );
-      if (!isEqual(documentMetadata.selectedCases, casesWithAPartySelected)) {
+      if (
+        !isEqual(
+          sortBy(documentMetadata.selectedCases),
```
Comment (sutt0n) [inline shared/src/business/entities/externalDocument/ExternalDocumentInformationFactory.js:267]: would this work with "101-19" and "102-18"?

File: web-client/src/presenter/sequences/completeDocumentSelectSequence.js [hunk 2]
```diff
@@ -25,6 +28,11 @@ export const completeDocumentSelectSequence = [
       generateTitleAction,
       setDocketNumberPropAction,
       setDefaultFileDocumentFormValuesAction,
+      isConsolidatedCaseAction,
+      {
+        no: [],
+        yes: [unset(state.form.partyPrimary)],
```
Comment (sutt0n) [inline web-client/src/presenter/sequences/completeDocumentSelectSequence.js:34]: should this be in its own action?

**Review by sutt0n — APPROVED** [review]
looks good, just a few questions

File: shared/src/business/entities/externalDocument/ExternalDocumentInformationFactory.js (hunk 1 above)
Comment (wbyoko) [reply shared/src/business/entities/externalDocument/ExternalDocumentInformationFactory.js:267]: these sorts are just to make sure both arrays are in the same order before they are compared.

File: web-client/src/presenter/sequences/completeDocumentSelectSequence.js (hunk 2 above)
Comment (wbyoko) [reply web-client/src/presenter/sequences/completeDocumentSelectSequence.js:34]: it always could be.

**Pull request merged**